C#/cs01/tests/14.cs

```diff
@@ -0,0 +1,45 @@
+using NUnit.Framework;
+using Brainchild.Curriculum.Solutions;
+namespace Brainchild.Curriculum
+{
+    [TestFixture]
+    public class Test14
+    {
+        [SetUp]
+        public void SetUp()
+        {
+        }
+        [Test]
```

Comment:
it will be good ,if you can add tests to  check the returning values from normalstrategy and happyhourstrategy classes,it will help when additional modification in program needed in future.

Reply:
Don't we already have 2 tests to check both HappyHour case and Normal case

Reply:
i mean separately for each classes